Modules/Devices/src/Devices.Application/DTOs/IdentityDeletionProcessAsSupportDTO.cs

```diff
@@ -0,0 +1,25 @@
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+
+namespace Backbone.Modules.Devices.Application.DTOs;
+
+public class IdentityDeletionProcessAsSupportDTO
```

Comment:
I thought that it's a good idea to have a separate one that clearly resembles that it's used as Support, but yea, now I see what was the problem. I should have used the already existing one. Changes made.

Reply:
There are actually reasons to use a separate one. But since currently there are no differences, it's okay to use the same, I think. Especially because there will be some properties added soon, and I want to avoid forgetting to add them to one of the DTOs.